quisp/modules/QRSA/HardwareMonitor/HardwareMonitor.cc

```diff
@@ -284,94 +354,91 @@ void HardwareMonitor::finish() {
         GOD_Z_pair_total++;
       } else if ((it->second.my_GOD_clean == 'Y' && it->second.partner_GOD_clean == 'F') || (it->second.my_GOD_clean == 'F' && it->second.partner_GOD_clean == 'Y')) {
         GOD_Y_pair_total++;
-      }
+      }  // end if
```

Comment:
do we need this comment? the "if" body is not so big. I don't think it is needed.